warp/tests/geometry/test_mesh.py

```
@@ -198,6 +198,40 @@ def query_ray_kernel(
+    mesh_id: wp.uint64,
+):
+    start = wp.vec3(0.1, 0.2, 0.3)
+    dir = wp.normalize(wp.vec3(-1.2, 2.3, -3.4))
+    expected_t = 0.557828
+
+    t = float(0.0)
+    bary_u = float(0.0)
+    bary_v = float(0.0)
+    sign = float(0.0)
+    normal = wp.vec3(0.0, 0.0, 0.0)
+    face = int(0)
```

Ruff (0.14.5)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)